### ISSUE TITLE
Support for U+2011 NON-BREAKING HYPHEN

### ISSUE THREAD

**Issue by valtlai** [adobe-fonts/source-sans-pro#72]
It would be useful to have the U+2011 NON-BREAKING HYPHEN character in Source Sans Pro. It looks like a hyphen, but programs  don’t wrap immediately before or after it.

More details: https://en.wikipedia.org/wiki/Hyphen#Nonbreaking_hyphens

---

Example:

With hyphen-minus:

```
Lorem ipsum dolor sit UTF-
8 amet
```

With non-breaking hyphen:

```
Lorem ipsum dolor sit
UTF-8 amet
```

**Comment by erniemarch** [comment]
Checked in Roman version 2.034 and Italic version 1.084. This is now present.

**Comment by miguelsousa** [comment]
Fixed in version 2.040.
